addon-test-support/-private/filter-test-modules.js

```diff
@@ -0,0 +1,18 @@
+
+/**
+ * Returns a list of test modules that match with the given module path filter.
+ *
+ * @param {*} modules
+ * @param {*} modulePathFilter
+ */
+export default function filterTestModules(modules, modulePathFilter) {
+  const tests = [];
+
+  modules.forEach((module) => {
+    if (module.includes(modulePathFilter)) {
```

Comment:
Question, Can regex be supported?

Reply:
current implementation doesn't but I will work on supporting regex as well